README.md

```diff
@@ -583,6 +583,66 @@ _You can enable the following settings in Xcode by running [this script](resourc
     listingUrgencyBookedShortRowContent(),
   ]
   ```
+  
+* <a id='multi-line-conditions'></a>(<a href='#multi-line-conditions'>link</a>) **Multi-line conditional statements should break after the leading keyword.** Indent each individual statement by [2 spaces](https://github.com/airbnb/swift#spaces-over-tabs). Put the open curly brace on the next line. In `guard` statements, place the `else` keyword on the same line as the open curly brace. [![SwiftFormat: wrapArguments](https://img.shields.io/badge/SwiftFormat-wrapArguments-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapArguments) [![SwiftFormat: wrapMultilineStatementBraces](https://img.shields.io/badge/SwiftFormat-wrapMultilineStatementBraces-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapMultilineStatementBraces) [![SwiftFormat: elseOnSameLine](https://img.shields.io/badge/SwiftFormat-elseOnSameLine-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#elseonsameline)
+
+  <details>
+    
+  #### Why?
+  Breaking after the leading keyword resets indentation to the standard [2-space grid](https://github.com/airbnb/swift#spaces-over-tabs),
+  which helps avoid fighting Xcode's <kbd>^</kbd> + <kbd>I</kbd> indentation behavior.
+
+  ```swift
+  // WRONG
+  if let galaxy = galaxy,
+    galaxy.name == "Milky Way" // Indenting by two spaces fights Xcode's ^+I indentation behavior
+  { … }
+  
+  // WRONG
+  guard let galaxy = galaxy,
+        galaxy.name == "Milky Way" // Variable width indentation (6 spaces)
+  else { … }
+  
+  // WRONG
+  if 
+    let galaxy = galaxy,
+    galaxy.name == "Milky Way" {
+    // This line blends in with the list of conditions  
+  }
+  
+  // WRONG
+  guard 
+    let galaxy = galaxy,
+    galaxy.name == "Milky Way" else // The else keyword should sit on the same line as the open brace
+  { … }
+
+  // WRONG
+  guard let earth = unvierse.find(
+    .planet,
+    named: "Earth"),
+    earth.isHabitable // Blends in with previous condition's method arguments
+  else { … }
+  
+  // RIGHT
```

Comment:
Is it still possible to write a one line guard statement if it all fits on one line?

```swift
guard myConditional else { fatalError() }
```

Reply:
Yes! This rule is only intended to apply to statements that wrap and span multiple lines.

Reply:
It could be nice to put in a single line statement to show that's OK.